src/mpid/ch4/src/ch4_rma.h

```diff
@@ -462,10 +462,8 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_put_safe(const void *origin_addr,
                             MPI_DATATYPE_NULL, target_rank, target_disp, target_count,
                             target_datatype, MPI_OP_NULL, win, NULL, NULL);
 #else
-    MPID_THREAD_CS_ENTER(VCI, MPIDI_VCI(0).lock);
     mpi_errno = MPIDI_put_unsafe(origin_addr, origin_count, origin_datatype,
```

Comment:
can we update the names of these "unsafe" functions at this point? "unsafe" no longer seems accurate.

Reply:
Sure. Not just update the names, I think we should merge the previous `_safe` and `_unsafe` variations. Let me add a commit update that.